webapp/platform/client/src/websocket.ts

```diff
@@ -26,6 +28,8 @@ const defaultWebSocketClientConfig: WebSocketClientConfig = {
     newWebSocketFn: (url: string) => {
         return new WebSocket(url);
     },
+    clientPingEnabled: true,
```

Comment:
What is the objective of this config setting if it's always true? I don't see this being configured from the server. :thinking:

Reply:
This was originally used for tests, but you're right - we don't need this anymore. Removed :+1: